src/components/TodosToggler/TodosToggler.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const TodosToggler = ({ onToggleTodosStatus, activeTodosLength }) => (
+  <>
+    <input
+      checked={activeTodosLength === 0}
+      type="checkbox"
+      id="toggle-all"
+      className="toggle-all"
+      onChange={(e) => {
+        onToggleTodosStatus(e);
+      }}
```

Comment:
```suggestion
      onChange={onToggleTodosStatus}
```
there is no need to make things complicated